scripts/linux/connectivityTest.sh

```diff
@@ -0,0 +1,258 @@
+    docker logs loadGen1 || true
+    
+    print_highlighted_message 'LoadGen2 LOGS'
+    docker logs loadGen2 || true
+}
+
+function process_args() {
+    print_highlighted_message 'Process arguments'
+    saveNextArg=0
+    for arg in "$@"
+    do
+        if [ $saveNextArg -eq 1 ]; then
+            E2E_TEST_DIR="$arg"
+            saveNextArg=0
+        elif [ $saveNextArg -eq 2 ]; then
+            RELEASE_LABEL="$arg"
+            saveNextArg=0
+        elif [ $saveNextArg -eq 3 ]; then
+            ARTIFACT_IMAGE_BUILD_NUMBER="$arg"
+            saveNextArg=0
+        elif [ $saveNextArg -eq 4 ]; then
+            CONTAINER_REGISTRY="$arg"
+            saveNextArg=0
+        elif [ $saveNextArg -eq 5 ]; then
+            CONTAINER_REGISTRY_USERNAME="$arg"
+            saveNextArg=0
+        elif [ $saveNextArg -eq 6 ]; then
+            CONTAINER_REGISTRY_PASSWORD="$arg"
+            saveNextArg=0
+        elif [ $saveNextArg -eq 7 ]; then
+            IOTHUB_CONNECTION_STRING="$arg"
+            saveNextArg=0
+        elif [ $saveNextArg -eq 8 ]; then
+            EVENTHUB_CONNECTION_STRING="$arg"
+            saveNextArg=0
+        elif [ $saveNextArg -eq 9 ]; then
+            EVENT_HUB_CONSUMER_GROUP_ID="$arg"
+            saveNextArg=0
+        elif [ $saveNextArg -eq 10 ]; then
+            LOADGEN_MESSAGE_FREQUENCY="$arg"
+            saveNextArg=0            
+        else
+            case "$arg" in
+                '-h' | '--help' ) usage;;
+                '-testDir' ) saveNextArg=1;;
+                '-releaseLabel' ) saveNextArg=2;;
+                '-artifactImageBuildNumber' ) saveNextArg=3;;
+                '-containerRegistry' ) saveNextArg=4;;
+                '-containerRegistryUsername' ) saveNextArg=5;;
+                '-containerRegistryPassword' ) saveNextArg=6;;
+                '-iotHubConnectionString' ) saveNextArg=7;;
+                '-eventHubConnectionString' ) saveNextArg=8;;
+                '-eventHubConsumerGroupId' ) saveNextArg=9;;
+                '-loadGenMessageFrequency' ) saveNextArg=10;;
+                '-cleanAll' ) CLEAN_ALL=1;;
+                * ) usage;;
+            esac
+        fi
+    done
+
+    # Required parameters
+    [[ -z "$RELEASE_LABEL" ]] && { print_error 'Release label is required.'; exit 1; }
+    [[ -z "$ARTIFACT_IMAGE_BUILD_NUMBER" ]] && { print_error 'Artifact image build number is required'; exit 1; }
+    [[ -z "$CONTAINER_REGISTRY_USERNAME" ]] && { print_error 'Container registry username is required'; exit 1; }
+    [[ -z "$CONTAINER_REGISTRY_PASSWORD" ]] && { print_error 'Container registry password is required'; exit 1; }
+    [[ -z "$IOTHUB_CONNECTION_STRING" ]] && { print_error 'IoT hub connection string is required'; exit 1; }
+    [[ -z "$EVENTHUB_CONNECTION_STRING" ]] && { print_error 'Event hub connection string is required'; exit 1; }
+
+    echo 'Required parameters are provided'
+}
+
+function run_connectivity_test() {
```

Comment:
Are there any "connectivity test" specific parts in here?

Reply:
what do you mean "connectivity test" specific parts?  This is 1st deployment to deploy edgeAgent, edgeHub and 2 loadGen modules; will progressively add other modules to deployment.